shopify-features/models/profiles.yaml

```diff
@@ -493,6 +534,10 @@ models:
     
       features:
         - days_since_last_seen
+        - total_units_across_transactions
```

Comment:
Lets not make these changes. Keep the changes only limited to the timestamp macro.